src/pages/upload/index.tsx

```diff
@@ -6,17 +6,43 @@ import fileExtension from 'file-extension'
 import {find} from 'lodash'
 import {GetServerSideProps} from 'next'
 import {getAbilityFromToken} from 'server/ability'
+import groq from 'groq'
+import {sanityClient} from 'utils/sanity-client'
+import {useViewer} from 'context/viewer-context'
 
 const SIGNING_URL = `/api/aws/sign-s3`
 
 type FileUpload = {file: File; percent: number; message: string}
 
+type Instructor = {
+  externalId: string
+  person: {
+    _id: string
+    name: string
+    image: string
+  }
+}
+
 export const getServerSideProps: GetServerSideProps = async function ({req}) {
   const ability = await getAbilityFromToken(req.cookies[ACCESS_TOKEN_KEY])
 
   if (ability.can('upload', 'Video')) {
+    const instructorQuery = groq`
+      *[_type == 'collaborator' && role == 'instructor'][]{
+        'person': person-> {
+            _id,
+            name,
+            'image': image.url,
+          },
+        externalId,
+      }`
+
+    const instructors: Instructor[] = await sanityClient.fetch(instructorQuery)
```

Comment:
Load collaborators that are instructors from Sanity and pass into `/upload` as a prop.